langchips-project/Models/Entry.cs

```diff
@@ -8,29 +9,27 @@ namespace langchips_project.Models
 {
     internal class Entry
```

Comment:
Proponuję nową klasę o roboczej nazwie Tranlation
{
Id,
Language,
Value
}

Entry
{
Id,
Translation Translation1,
Translation Translation2,
}

Reply:
Nie za bardzo wiem jak podejsc do tematu teraz gdy nad tym pracuję. Na ten moment mam dicitonary jako jeden zestaw oraz entry nadal jako slowko, tlumaczenie i odpowiednie jezyki. W tej klasie, Translation pewnie mielibysmy jakis "paiting" entries i to jego powinnismy wtedy w bazie danych zapisac? Nie wiem jak ugryzc temat.